repository-data/webfiles/src/main/resources/site/freemarker/publicationsystem/dataset.ftl

```diff
@@ -1,120 +1,173 @@
 <#ftl output_format="HTML">
 <#include "../include/imports.ftl">
 <#include "./macro/structured-text.ftl">
-<@hst.setBundle basename="publicationsystem.headers,publicationsystem.labels"/>
+<@hst.setBundle basename="publicationsystem.labels,publicationsystem.headers"/>
 
+<#assign formatFileSize="uk.nhs.digital.ps.directives.FileSizeFormatterDirective"?new() >
```

Comment:
This is included in the imports.ftl file, so could remove to avoid duplication.

Reply:
Thanks @ngta1, I'll remove that line and will update my PR.